index.html

```diff
@@ -24,6 +24,90 @@ <h1>Felipe Haybar</h1>
             <a class="textBoton" href="#">Start collaboration</a>
         </button>
         <div class="imagebox"></div>
-    </section>
+        <section class="projects">
```

Comment:
- [ ] KIndly display this main section as `grid `according to the requirement. 😄

Reply:
Hi Oye, 
grid applied on the projects section and about section, look better.
Thank you